src/UnitTestExecutor.cpp

```diff
@@ -31,6 +33,7 @@ void UnitTestExecutor::executeFilterTests(const GUID& guid) {
 	Executes all tests across all filters.
 */
 void UnitTestExecutor::executeAllTests() {
+	logger.info(L"Executing all tests.");
```

Comment:
Spíš něco jako "Executing all tests across all filters", aby bylo vidět, že to je něco jiného, než v GenericUnitTesteru.